woocommerce/woocommerce-ios
# Change badge view in order products list text color

Closes #12628

## Description


Noticed that in dark mode badge view text color is to dark. Since the background color of badg view is black I set the text color to plain white.

## Testing instructions


- Open orders
- Add product(s) to the order
- Check that the number if badge is readable
- Test it in both light and dark mode

## Screenshots


| Before      | After |
| ----------- | ----------- |
| ![Simulator Screenshot - iPad Pro (11-inch) (4th generation) - 2024-05-02 at 11 22 22](https://github.com/woocommerce/woocommerce-ios/assets/6242034/0b1f0891-f07f-413d-8f9b-5f7cf549c1d7)      | ![Simulator Screenshot - iPad Pro (11-inch) (4th generation) - 2024-05-02 at 11 20 50](https://github.com/woocommerce/woocommerce-ios/assets/6242034/e8031029-9ed8-4215-99d6-1c689c3b7c8d)       |

---
